var/spack/repos/builtin/packages/elfutils/package.py

```diff
@@ -114,6 +116,11 @@ def configure_args(self):
         else:
             args.append("--without-lzma")
 
+        if "+zstd" in spec:
```

Comment:
FYI.  I realize this addition is consistent with the style used for other args, but thought it useful to point to one of the AutotoolsPackage helpers for these types of arguments (see https://spack.readthedocs.io/en/latest/build_systems/autotoolspackage.html#activation-value).

Reply:
I have updated this part, it makes the code a lot cleaner. Thanks for the information.